components/DonationV2/donationV2.tsx

```diff
@@ -0,0 +1,42 @@
+export default function DonationV2() {
+  const imageUrl = 'https://source.unsplash.com/630x441/';
+  return (
+    <section>
+      <div className="container px-6 py-10 mx-auto">
+        <h1 className="text-3xl font-bold text-[#1A1A1A]">
+          Поддержать сообщество
+        </h1>
+
+        <div className="mt-8 lg:-mx-6 lg:flex lg:items-center">
+          <picture className="w-full lg:mx-6 lg:w-1/2 h-72 lg:h-96">
+            <source srcSet={imageUrl} type="image/webp" />
+            <img
+              className="object-cover w-full h-72 lg:h-96"
+              src={imageUrl}
+              alt=""
+            />
```

Comment:
Any reason why do you use **img** tag instead of **next/image**? (https://nextjs.org/docs/api-reference/next/image)

**alt** property is empty?